types/textfit/index.d.ts

```diff
@@ -0,0 +1,66 @@
+// Type definitions for textfit 2.4
+// Project: https://github.com/STRML/textFit
+// Definitions by: Muhun Kim <https://github.com/x86chi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function textFit(element: HTMLElement, options: textFit.TextFitOption): void;
```

Comment:
```suggestion
declare function textFit(element: HTMLElement, options?: textFit.TextFitOption): void;
```
this is optional:
https://github.com/STRML/textFit/blob/5a102903acdacfecbee6bfd88fde1a1413964339/textFit.js#L48